### ISSUE TITLE
"Module" name in generated JS is misleading

### ISSUE THREAD

**Issue by iliakan** [emscripten-core/emscripten#18727]
The right name would be "Instance".

Because Module in WASM is something else (compiled program code), and what the script is doing - it creates an instance.

What do you think?

**Comment by sbc100** [comment]
Yes, sadly that term Module here predates WebAssembly itself.   If we could change it easily I agree that we should.   Doing so without breaking too many folks is that hard part.

As you have noticed the term is also used in a lot of places so changing all them would be a considerable effort.

**Comment by iliakan** [comment]
Yes, I agree it's not eaily doable within the current Emscripten major version. 
Maybe sometime there'll be a new major, and then we'll remember about it =)
